cudamapper/src/main.cu

```diff
@@ -101,6 +105,12 @@ int main(int argc, char* argv[])
         exit(1);
     }
 
+    if (max_cached_memory <= 0)
+    {
+        std::cerr << "-m / --max-cached-memory must be non-zero" << std::endl;
```

Comment:
I guess this should read "must be larger than zero".

Reply:
done